File: application/src/main/java/ru/artemev/application/model/enums/EmploymentStatus.java
package ru.artemev.application.model.enums;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Schema(description = "Рабочий статус")
@AllArgsConstructor
@Getter
public enum EmploymentStatus {
  UNEMPLOYED("Безработный"),
  SELF_EMPLOYED("Частный предприниматель"),
  EMPLOYED("Рабочий"),
  BUSINESS_OWNER("Владелец бизнеса");

  private final String description;
}

File: gateway/src/main/kotlin/ru/artemev/gateway/model/enums/EmploymentStatus.kt
package ru.artemev.gateway.model.enums

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "Рабочий статус")
enum class EmploymentStatus(val description: String) {
    UNEMPLOYED("Безработный"),
    SELF_EMPLOYED("Частный предприниматель"),
    EMPLOYED("Рабочий"),
    BUSINESS_OWNER("Владелец бизнеса")
}
